openapi/components/schemas/AuthorizationsProperties.yaml

```diff
@@ -0,0 +1,19 @@
+authorizations:
+  type: array
+  items:
+    type: string
+    enum:
+      - None
+      - BasicAuth
+      - BearerAuth
+      - PassportAuth
+  description: >-
+    An Optional list of support authorization types. More than one can be supported and tried in sequence.
+    Defaults to `None` if empty or missing.
+passport_issuers:
```

Comment:
Additionally, would it be useful/good to point to the passport broker URL?

Reply:
I think this is hard. Each consuming system of DRS likely has its own Broker (e.g. Terra has its own, SBG probably does too). How would the DRS server know the list? I think where the caller gets their passport should stay out of this spec. Said another way, I think this spec should be able to say what passports it needs but not where or how to get them.